diff -up openttd-1.8.0/src/language.h.omv~ openttd-1.8.0/src/language.h
--- openttd-1.8.0/src/language.h.omv~	2018-04-09 21:27:05.077684321 +0200
+++ openttd-1.8.0/src/language.h	2018-04-09 21:30:22.441725968 +0200
@@ -105,7 +105,7 @@ extern LanguageList _languages;
 extern const LanguageMetadata *_current_language;
 
 #ifdef WITH_ICU_SORT
-extern Collator *_current_collator;
+extern U_ICU_NAMESPACE::Collator *_current_collator;
 #endif /* WITH_ICU_SORT */
 
 bool ReadLanguagePack(const LanguageMetadata *lang);
diff -up openttd-1.8.0/src/strings.cpp.omv~ openttd-1.8.0/src/strings.cpp
--- openttd-1.8.0/src/strings.cpp.omv~	2018-04-09 21:30:53.076935797 +0200
+++ openttd-1.8.0/src/strings.cpp	2018-04-09 21:31:30.475109406 +0200
@@ -52,7 +52,7 @@ const LanguageMetadata *_current_languag
 TextDirection _current_text_dir; ///< Text direction of the currently selected language.
 
 #ifdef WITH_ICU_SORT
-Collator *_current_collator = NULL;               ///< Collator for the language currently in use.
+U_ICU_NAMESPACE::Collator *_current_collator = NULL;               ///< Collator for the language currently in use.
 #endif /* WITH_ICU_SORT */
 
 static uint64 _global_string_params_data[20];     ///< Global array of string parameters. To access, use #SetDParam.
@@ -1795,7 +1795,7 @@ bool ReadLanguagePack(const LanguageMeta
 
 	/* Create a collator instance for our current locale. */
 	UErrorCode status = U_ZERO_ERROR;
-	_current_collator = Collator::createInstance(Locale(_current_language->isocode), status);
+	_current_collator = U_ICU_NAMESPACE::Collator::createInstance(U_ICU_NAMESPACE::Locale(_current_language->isocode), status);
 	/* Sort number substrings by their numerical value. */
 	if (_current_collator != NULL) _current_collator->setAttribute(UCOL_NUMERIC_COLLATION, UCOL_ON, status);
 	/* Avoid using the collator if it is not correctly set. */
